i18n/locales/en.ts

```diff
@@ -1192,19 +1192,59 @@ const translations = {
+        op_checksig_heading: 'OP_CHECKSIG',
+        op_checksig_paragraph_one:
+          'OP_CHECKSIG in Bitcoin script is critically important to ensuring that the correct private key is able to spend a given transaction. In almost every Bitcoin script there will be an OP_CHECKSIG to ensure that the person attempting to spend the bitcoin is able to do so with the given key. You can read more about some of these OP_CODES and more <Link href="https://en.bitcoin.it/wiki/Script#Constants" target="_blank" className="underline">here</Link>.',
+      },
+      derive_message_six: {
+        transaction_parts_heading: 'Transaction Parts',
+        transaction_parts_one:
+          'Version: This version indicates how the the transaction will be organzied.',
+        transaction_parts_two:
+          'Number of inputs: The number of inputs in theis transaction.',
+        transaction_parts_three:
+          'Hash on input#0: This is the hash of the tx corresposing to input#0 subsequent inputs will go after part six.',
+        transaction_parts_four:
+          'Index of input#0: This is the index of the input#0 from the funding transaction as it could have many outputs.',
+        transaction_parts_five:
+          'Scriptsig: This is the scriptsig that signs the input for this tx.',
+        transaction_parts_six:
+          'Input#0 sequence: This is the sequence for the spending input.',
+        transaction_parts_seven:
+          'Number of outputs: This indicates how many outputs the tx should have.',
+        transaction_parts_eight:
+          'Output#0 value: This is the amount that should be spent to the first output in little-endian bytes.',
+        transaction_parts_nine:
+          'Output#0 scriptPubKey: This is the public key plus OP_CHECKSIG to ensure that the transaction is being spent correctly.',
+        transaction_parts_ten:
+          'Output#1 value: This is the amount that should be spent to the second output in little-endian bytes.',
+        transaction_parts_eleven:
+          'Output#1 scriptPubKey: This is the public key of the sender so that the change output will go back into their possesion.',
+        transaction_parts_twelve:
+          'Locktime: Amount of time that this tx should be locked before the coins can be spent.',
+      },
+      derive_message_seven: {
+        sighash_type_flag_heading: 'SigHash Type Flag',
+        sighash_type_flag_paragraph_one:
+          'SIGHASH flags are a mechanism in Bitcoin that define which parts of a transaction are included in the hash that is signed by a private key. Essentially, they determine the scope of commitment by the signer to specific parts of the transaction data. The SIGHASH flag is a single byte appended to each signature and can vary between inputs within the same transaction. There are several types of SigHash flags you can learn about <Link href="https://river.com/learn/terms/s/sighash-flag" target="_blank" className="underline">here</Link>.',
```

Comment:
```suggestion
          'SigHash flags are a mechanism in Bitcoin that define which parts of a transaction are included in the hash that is signed by a private key. Essentially, they determine the scope of commitment by the signer to specific parts of the transaction data. The SigHash flag is a single byte appended to each signature and can vary between inputs within the same transaction. There are several types of SigHash flags you can learn about <Link href="https://river.com/learn/terms/s/sighash-flag" target="_blank" className="underline">here</Link>.',
```

Opting for "SigHash" over "SIGHASH" because that's what chat.bitcoinsearch.xyz used in its answer to me :laughing: but I don't really care as long as it's consistent in this paragraph.